paper/symposium.md

```diff
@@ -0,0 +1,214 @@
+## The Unrollment Project: Exploring Algorithmic Bias in Higher Education
+
+This symposium will provide an overview and a discussion of our
+ongoing project exploring the use of predictive algorithms in higher
+education decisionmaking.  In this project team members use an open
+source platform to provide transparency around how predictive
+algorithms in higher education are estimated and how decisions made on
+the basis of these algorithms might affect currently underserved
+students in higher education. Predicitve algorithms are any predefined 
+computational procedure that provides a likely outcome for a given individual
+based on the individual's characteristics. For instance, using coefficients from 
+a regression to predict the likely earnings for an individual based on their level 
+of education would be an example of using a predictive algorithm. Predictive 
+algorithms must by necessity be based on observed relationships from the past.  
+Predictive algorithms have taken on an
+important role in decisionmaking in higher education, and have clear
+implications for participation for all (Eduventures, 2013; Ekowo &
+Palmer 2016). This project seeks to demistify how these algorithms
+work and to create a more open discussion about how these predictive
+models might affect participation in higher education.
```

Comment:
bias isn't mentioned until paragraph 3. this sort of "buries the lead" so last sentence of first paragraph should say that the project investigates their predictive power and potential biases or something like that.

Reply:
Done